common/middleware/middleware.go

```diff
@@ -31,3 +33,12 @@ func Merge(middlesware ...Interface) Interface {
 		return next
 	})
 }
+
+// Gorilla Router with sensible defaults, namely:
```

Comment:
I think this will make the linter fail (the comment needs to start with `NewRouter()`). Did you get to run `make lint` locally?